packages/botonic-react/src/components/button.jsx

```diff
@@ -100,20 +125,22 @@ export const Button = props => {
         )
 
     return (
-      <StyledButton
-        theme={theme}
-        onMouseEnter={() => setHover(true)}
-        onMouseLeave={() => setHover(false)}
-        onClick={e => handleClick(e)}
-        style={{
-          ...buttonStyle,
-          color: buttonTextColor,
-          backgroundColor: buttonBgColor,
-        }}
-        bottom={props.bottomRadius}
-      >
-        {props.children}
-      </StyledButton>
+      enabled && (
```

Comment:
We should check here the `diableButtons` value too, right? If we don't do it here, even if the feature is not enabled, we could hide buttons anyway.